docs/SuperSIDonLinux.md

```diff
@@ -98,7 +97,7 @@ From /home/pi:
     # Activate the newly created virtual environment
     $ source supersid-env/bin/activate
     # Install latest versions of package tools
-    $ python -m pip install -U pip wheel setuptools
+    $ python3 -m pip install -U pip wheel setuptools

```

Comment:
In my experience this change is not needed. Once you create the virtual env with python3, and activate the environment, "python" will run python3. The virtual environment should be setting the version of python that is used in that environment.

Reply:
I have not been aware of this effect. Will add the required command to install venv on Ubuntu and an explanation about python vs. python3 after the activation of the virtual environment.